client/src/partials/vouchers/simple.js

```diff
@@ -0,0 +1,105 @@
+angular.module('bhima.controllers')
+.controller('SimpleJournalVoucherController', SimpleJournalVoucherController);
+
+SimpleJournalVoucherController.$inject = [
+  'AppCache', 'VoucherService', '$translate', 'AccountService',
+  'CurrencyService', 'SessionService'
+];
+
+/**
+ * Simple Journal Vouchers
+ *
+ * This module implements simply journal vouchers, crafted specially for cash
+ * transactions, but useable in any generic transactions that only require two
+ * Posting Journal lines.  It allows users to quickly create transactions by
+ * specifying two accounts and an amount to transfer between them.
+ *
+ * @constructor
+ *
+ * @todo - Implement caching mechanism for incomplete forms (via AppCache)
+ * @todo - Implement Voucher Templates to allow users to save pre-selected
+ * forms (via AppCache and the breadcrumb component).
+ */
+function SimpleJournalVoucherController(AppCache, Vouchers, $translate, Accounts, Currencies, Session) {
+  var vm = this;
+
+  // cache to save work-in-progress data and pre-fabricated templates
+  var cache = AppCache('JournalVouchers');
+
+  // bread crumb paths
+  vm.paths = [{
+    label : $translate.instant('VOUCHERS.SIMPLE.TITLE'),
+    current: true
+  }];
+
+  // bind the startup method as a reset method
+  vm.reset = startup;
+  vm.submit = submit;
+
+  // load the list of accounts
+  Accounts.list()
+  .then(function (accounts) {
+    vm.accounts = accounts;
+  });
+
+  // load the available currencies
+  Currencies.read()
+  .then(function (currencies) {
+
+    // format human readable labels
+    currencies.forEach(function (currency) {
+      currency.label = Currencies.format(currency.id);
+    });
+
+    // bind the currencies to the view
+    vm.currencies = currencies;
+  });
+
+  /** run the module on startup and refresh */
+  function startup() {
+
+    // delete any state indicators (if they exist)
+    delete vm.created;
+
+    // this is the voucher form to be submitted
+    vm.voucher = {};
+
+    // set up default voucher values
+    vm.voucher.date = new Date();
+    vm.voucher.description =
+      $translate.instant('VOUCHERS.SIMPLE.DESCRIPTION');
```

Comment:
I agree.

My fear with `placeholder` is that users will not fill in a proper transaction description, so I wanted to pre-populate with text as guidelines for them to fill it out.  Something like "Transaction made by jniles on {{ date }} for {{ reason }}".  However, since we cannot template in values to the translated text, this PR was undecided and left the whole thing in a pretty broken state -- sorry about that.

I'll switch to using `placeholder` until we get some real data.  Perhaps an [A/B Test](https://en.wikipedia.org/wiki/A/B_testing) will be able to determine if `placeholder` is enough or not.